app/Http/Requests/BaseApiRequest.php

```diff
@@ -287,7 +291,7 @@ protected function authorizePhotosWrite(EloquentCollection $photos): bool
 	 */
 	protected function authorizePhotosWriteByIDs(array $photoIDs): bool
 	{
-		return $this->photoAuthorisationProvider->areEditableByIDs($photoIDs);
+		return Gate::check(UserPolicy::UPLOAD, User::class) && $this->photoPolicy->editByID(Auth::user(), $photoIDs);
```

Comment:
```suggestion
		return $this->photoPolicy->editByID(Auth::user(), $photoIDs);
```

See comment for `authorizeAlbumsWriteByIDs` above.